liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -282,6 +282,22 @@ info.server.stop.noexist=CWWKM2170I: Server {0} cannot be stopped because it doe
 info.server.stop.noexist.explanation=A required server does not exist.
 info.server.stop.noexist.useraction=No action is required.
 
-error.install.app.missing=CWWKM2170E: The application for this project does not exist and cannot be installed.
+error.install.app.missing=CWWKM2171E: The application for this project does not exist and cannot be installed.
 error.install.app.missing.explanation=The application either needs to be built, or there is no application code in this project. 
 error.install.app.missing.useraction=Run the goals to create the application to be installed.
+
+info.loose.application.not.supported=CWWKM2172I: Loose application configuration is not supported for packaging type {0}. The project artifact will be installed as is.
+info.loose.application.not.supported.explanation=Loose application configuration is only supported for web application with packaging type in this version of liberty-maven-plugin. 
```

Comment:
"with" isn't working for me in the explanation.
Loose application configuration is only supported for **the web application packaging type** in this version of liberty-maven-plugin. 
or maybe:
Loose application configuration is only supported for **the web packaging type** in this version of liberty-maven-plugin.